pegasus/src/curriculum_router.rb

```diff
@@ -395,6 +395,24 @@ class CurriculumRouter < Pegasus::Base
     return send_file(path)
   end
 
+  get '/curriculum/advocacy/*' do |file|
```

Comment:
hm, if you're defining a new route anyway, can you just make it a url without /curriculum in it? or does something else depend on the curriculum in the url?